lib/google_crawler/accounts.ex

```diff
@@ -349,4 +349,15 @@ defmodule GoogleCrawler.Accounts do
       {:error, :user, changeset, _} -> {:error, changeset}
     end
   end
+
+  def authenticate_and_generate_api_token(email, password) do
+    case get_user_by_email_and_password(email, password) do
+      %User{} = user ->
+        {_token, user_token} = UserToken.build_api_token(user)
+        {:ok, Repo.insert!(user_token)}
```

Comment:
Maybe we could do

```
{token, user_token} = UserToken.build_api_token(user)
Repo.insert!(user_token)
{:ok, token}
```

kinda similar to `generate_user_session_token` cause we only need to use token

Reply:
But in my controller, I need whole UserToken struct for rendering json
```
       case Accounts.authenticate_and_generate_api_token(email, password) do
        {:ok, token} ->
          conn
          |> put_view(GoogleCrawlerWeb.Api.UserTokenView)
          |> render("show.json", data: token)
```